website/templates/news.html

```diff
@@ -0,0 +1,34 @@
+<!--
+Copyright 2023 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<html>
+<head>
+  {{template "head.html" .}}
+  <title>{{.Title}}</title>
+  <link rel="stylesheet" href="/assets/css/common.css">
+  <link rel="stylesheet" href="/assets/css/blog.css">
+</head>
+<body>
+{{template "header.html" .}}
+<div class="page_blog">
```

Comment:
Use `page` instead of `page_blog`?

Reply:
Nice, much better. I also removed it from contact.html.